src/backends/mysql/session.cpp

```diff
@@ -368,7 +447,7 @@ void hard_exec(MYSQL *conn, const string & query)
     if (0 != mysql_real_query(conn, query.c_str(),
             static_cast<unsigned long>(query.size())))
     {
-        //throw soci_error(mysql_error(conn));		
```

Comment:
Generally, PRs should not include whitespace and formatting change, unless necessary or intentional.

This is single place, so minor issue, and actually it's good to trim trailing whitespaces, so no need to revert it.

Reply:
I understand the code modifying policy. Thank you for mention it :)